system-contract-testing/hardhat.config.js

```diff
@@ -5,12 +5,7 @@ require("@openzeppelin/hardhat-upgrades");
 require("@nomicfoundation/hardhat-chai-matchers");
 require("solidity-coverage");
 
-const {
-  OPERATOR_ID_A,
-  OPERATOR_KEY_A,
-  NETWORKS,
-  PRIVATE_KEYS,
-} = require("./utils/environment");
+const { NETWORKS, TESTNET_PRIVATE_KEYS } = require("./utils/environment");
```

Comment:
Do you want to support running this against something else then testnet in the future?